<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="rodDataTable">
        <property name="minimumSize">
         <size>
          <width>511</width>
          <height>191</height>
         </size>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>L, м</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>A, см^2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>σ, МПа</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>E, МПа</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="leftTerm">
          <property name="text">
           <string>Заделка слева</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="rightTerm">
          <property name="text">
           <string>Заделка справа</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addRodButton">
          <property name="maximumSize">
           <size>
            <width>127</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Добавить стержень</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteRodButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>127</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Удалить стержень</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="focusedForces">
        <property name="maximumSize">
         <size>
          <width>347</width>
          <height>281</height>
         </size>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>F, ql</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="distributedForces">
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>q, Н/м</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="newFile"/>
    <addaction name="separator"/>
    <addaction name="saveFile"/>
    <addaction name="separator"/>
    <addaction name="saveFileAs"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newFile">
   <property name="text">
    <string>Создать новый файл</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saveFileAs">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Добавить стержень</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Удалить выбранный стержень</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Удалить все стержни</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Добавить заделку слева</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Добавить заделку справа</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>Удалить заделку слева</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Удалить заделку справа</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>Удалить все заделки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
